### PR TITLE
Symfony3.4

### PR DESCRIPTION
**Pull request by Alsciende**

### CODE REVIEW

**Comment by Alsciende** [comment]
@stopfstedt I upgraded the application to use Symfony 3.4. Can you check it out? It should work out of the box, just checkout the branch and run `make install`.

**Comment by stopfstedt** [comment]
beat me to it! :D

will kick the tires on it tomorrow night.

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+# Set parameters here that may be different on each deployment target of the app, e.g. development, staging, production.
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 parameters:
-    database_host:     127.0.0.1
-    database_port:     ~
-    database_name:     thronesdb
-    database_user:     root
+    database_host: 127.0.0.1
+    database_port: ~
+    database_name: symfony
```
Comment (stopfstedt) [inline app/config/parameters.yml.dist]: should be `thronesdb`.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
-A Symfony project created on January 31, 2018, 10:36 am.
+# Very quick guide on how to install a local copy
+
+This guide assumes you know how to use the command-line and that your machine has php and mysql installed.
+
+- install composer: https://getcomposer.org/download/
+- clone the repo somewhere
+- cd to it
+- run `composer install` (at the end it will ask for the database configuration parameters)
+- run `php app/console doctrine:database:create`
```
Comment (stopfstedt) [inline README.md]: change `php app/console` to `bin/console` everywhere as applicable.

**Comment by stopfstedt** [comment]
also: in the `README.md`, replace `php app/console` with `php bin/console`.

**Comment by stopfstedt** [comment]
currently blocked by this fatal error: 

```
Fatal error: require(): Failed opening required '/home/stefan/dev/projects/thronesdb/var/cache/prod/doctrine/orm/Proxies/__CG__AppBundleEntityCycle.php' (include_path='.:/usr/share/php') in /home/stefan/dev/projects/thronesdb/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 223
```

flushing cache/rebooting apache etc did not clear this problem. will try again with a fresh install in the morning, and see if this persists.

**Comment by stopfstedt** [comment]
otherwise, this looks good to me. did some click testing, and couldn't find any problems.

**Pull request merged**